async-query-core/src/main/java/org/opensearch/sql/spark/utils/SQLQueryUtils.java

```diff
@@ -38,13 +43,14 @@
  */
 @UtilityClass
 public class SQLQueryUtils {
+  private static final Logger logger = LogManager.getLogger(SQLQueryUtils.class);
```

Comment:
nit: We prefer `LOG` instead of `logger`, though I see some variation in this repository...